### PR TITLE
Updates to r4 CapabilityStatement

### PR DESCRIPTION
**Pull request by KevinCranmer**

### CODE REVIEW

File: base-r4/src/main/java/gov/va/api/health/informational/r4/capability/CapabilityUtilities.java [hunk 1]
```diff
@@ -123,6 +136,14 @@ public static CapabilityStatement initializeCapabilityStatementBuilder(
         && !capabilityStatementProperties.getDescription().isBlank()) {
       capabilityStatementBuilder.description(capabilityStatementProperties.getDescription());
     }
+    // Implementation is optional.
+    if ((capabilityStatementProperties.getImplementationUrl() != null
+            && !capabilityStatementProperties.getImplementationUrl().isBlank())
```
Comment (jblefkove-va) [inline base-r4/src/main/java/gov/va/api/health/informational/r4/capability/CapabilityUtilities.java:141]: `StringUtils.isNotBlank()`?

File: base-r4/src/main/java/gov/va/api/health/informational/r4/capability/CapabilityUtilities.java [hunk 2]
```diff
@@ -123,6 +136,14 @@ public static CapabilityStatement initializeCapabilityStatementBuilder(
...
+        || (capabilityStatementProperties.getImplementationDescription() != null
+            && !capabilityStatementProperties.getImplementationDescription().isBlank())) {
```
Comment (jblefkove-va) [inline base-r4/src/main/java/gov/va/api/health/informational/r4/capability/CapabilityUtilities.java:143]: Can we move this check inside `capabilityImplementation()`, and let it return `null` if necessary?

**Comment by KevinCranmer** [comment]
I overlooked some `info` items that we could correct. Going to try and make those updates now

**Comment by KevinCranmer** [comment]
https://github.com/department-of-veterans-affairs/health-apis-hdr-fhir-query/pull/37
https://github.com/department-of-veterans-affairs/health-apis-data-query/pull/1111

File: conformance-unifier/src/main/resources/application.properties [hunk 3]
```diff
@@ -42,14 +42,16 @@ stu3.capability.security.description=http://docs.smarthealthit.org/
 # R4 Unified Metadata
 r4.capability.id=health-api-fhir-r4-capability
 r4.capability.version=@jenkins-build-version@
-r4.capability.name=API Management Platform | Health | FHIR R4
+r4.capability.name=api-management-platform-health-fhir-r4
```
Comment (KevinCranmer) [inline conformance-unifier/src/main/resources/application.properties:45]: Curious what you think of this name. Is there any risk in changing this? Here's the inferno issue:

> CapabilityStatement/health-api-fhir-r4-capability: CapabilityStatement: Rule cpb-0: 'Name should be usable as an identifier for the module by machine processing applications such as code generation' Failed

Comment (jblefkove-va) [reply conformance-unifier/src/main/resources/application.properties:45]: I see no risk in changing it

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**